simple_rpc/simple_rpc.py

```diff
@@ -24,17 +25,16 @@ def __init__(self, device, baudrate=9600, wait=1, autoconnect=True):
         :arg int wait: Time in seconds before communication starts.
         :arg bool autoconnect: Automatically connect.
         """
-        self._device = device
         self._wait = wait
 
-        self._connection = serial_for_url(device)
-        self._connection.baudrate = baudrate
+        self._connection = serial_for_url(
+            device, do_not_open=True, baudrate=baudrate)
```

Comment:
Not opening device in the first place, instead of opening and closing (old line 37).
`baudrate` can be passed directly, no need to make it a member variable.

Reply:
Nice.